combat-trainer.lic

```diff
@@ -2259,16 +2261,33 @@ class AttackProcess
       game_state.loaded = false
       waitrt?
     elsif game_state.loaded
-      actions = game_state.next_aim_action.split(';')
-      actions.each do |action|
-        first_word = action.split(' ')[0].downcase
-        matches = [/^You /]
-        matches = ['You get', 'You draw out', 'You pick up', "You're already holding that"] if @get_actions.include?(first_word)
-        matches = ['Roundtime',"You aren't close enough to attack"] if @rt_actions.include?(first_word)
-        matches = ['You put', 'You sheathe'] if @stow_actions.include?(first_word)
-        bput(action.strip, matches)
-        pause 0.25
-        waitrt?
+      if game_state.use_offhand_trainables?
```

Comment:
Rather than a setting, What about checking whether `@offhand_trainables` is set?

Reply:
That's what this done, but it's in the game_state. `use_offhand_trainables?` is a method that returns if the array is empty. I guess I could just make an accessor to @offhand_trainables in GameState.